src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct

```diff
@@ -144,10 +141,7 @@
 
       <Button guid="guidInteractiveWindowCmdSet" id="cmdidHistoryNext" priority="0x0300" type="Button">
         <Parent guid="guidInteractiveWindowCmdSet" id="ConsoleMenuGroup"/>
-        <Icon guid="ImageCatalogGuid" id="MoveDown" />
-        <!-- TODO (https://github.com/dotnet/roslyn/issues/5295)
         <Icon guid="ImageCatalogGuid" id="ListMembers" />
```

Comment:
You mean this ListMembers?  Does UX know that it's already an existing comment for statement completion?

<img width="636" alt="screen shot 2016-02-18 at 9 15 20 am" src="https://cloud.githubusercontent.com/assets/312835/13151852/e9f9bfc4-d620-11e5-97b1-0af5cd5691b4.png">